### ISSUE TITLE
Renaissance of diffusion models – do they really beat GANs? Stain style transfer and data augmentation for brain histology images.

### ISSUE THREAD

**Issue by Remi-Gau** [brainhackorg/global2022#135]
Added as an issue for book keeping

Source: https://www.brainhack-krakow.org/projects

Team Leaders: 

Monika Pytlarz, Sylwia Malec / [m.pytlarz@sanoscience.org](mailto:m.pytlarz@sanoscience.org?subject=Brainhack%20Krakow%202022)
/ [github octpsmon](https://github.com/octpsmon)

Abstract:

Assessment of brain tissue can be more precise by combining different histology stainings. Generating digital fluorescence histology would be beneficial, because of fewer artifacts and easier diagnostic recognition than on grayscale images. Staining style transfer is also widely used for the normalization task compensating for variability occurring during the sample preparation. The goal is to implement a transfer style (or data augmentation and transfer style) for generating histology images of different staining. Recent studies describing the renaissance of diffusion probabilistic models suggest that they can prove to be superior to variations of GANs within the transfer style and data augmentation. In the case of researching brain pathologies, we are facing the challenge of lacking publicly available big histology datasets and benchmarks. Data augmentation helps overcome the challenge of small sample size settings, improves the model prediction accuracy, and reduces data overfitting. The aim of the project is to compare these two competitive neural networks – the diffusion model vs GAN – and to determine the winner in the scope of the specified domain.
‍
List of materials:

Diffusion models:
1. Hiroshi Sasaki, Chris G. Willcocks, Toby P. Breckon. „UNIT-DDPM: Unpaired Image Translation with Denoising Diffusion Probabilistic Models”, 2021.
2. Yang Song, Stefano Ermon. „Generative Modeling by Estimating Gradients of the Data Distribution”, 2019.
3. Prafulla Dhariwal, Alex Nichol. „Diffusion Models Beat GANs on Image Synthesis”, 2021.

Cycle-GAN:
1. Jun-Yan Zhu, Taesung Park, Phillip Isola Alexei A. Efros. „Unpaired Image-to-Image Translation using Cycle-Consistent Adversarial Networks”, 2020.

Stain style transfer:
1. M. T. Shaban, C. Baur, N. Navab and S. Albarqouni, "Staingan: Stain Style Transfer for Digital Histological Images," 2019 IEEE 16th International Symposium on Biomedical Imaging (ISBI 2019), 2019, pp. 953-956, doi: 10.1109/ISBI.2019.8759152.
2. Tanishq Abraham, Andrew Shaw, Daniel O'Connor, Austin Todd, Richard Levenson. „Slide-free MUSE Microscopy to H&E Histology Modality Conversion via Unpaired Image-to-Image Translation GAN Models”. 2020.
3. Lee J-S, Ma Y-X. Stain Style Transfer for Histological Images Using S3CGAN. Sensors. 2022; 22(3):1044. https://doi.org/10.3390/s22031044.
4. Izadyyazdanabadi M, Belykh E, Zhao X, et al. „Fluorescence Image Histology Pattern Transformation Using Image Style Transfer”. Front Oncol. 2019;9:519. Published 2019 Jun 25. doi:10.3389/fonc.2019.00519.

‍
List of requirements for taking part in the project:

python programming, machine learning, deep learning skills; familiarity with generative adversarial networks, basic knowledge about above mentioned medical imaging modalities may be also useful
‍
Maximal allowed number of team members: 9
